spec/features/peep_spec.rb

```diff
@@ -0,0 +1,8 @@
+require_relative 'web_helper'
+
+feature 'User can post a peep' do
+  scenario 'Shows one peep on the page' do
+    post_peep
+    expect(page).to have_content 'Writing a peep'
+  end
+end
```

Comment:
It's good to see tests, but it'd be even better to see the tests pushed before they pass so you can prove your workflow is methodical. Remember: Failing test, push, passing test, push, refactor, push, repeat.